### ISSUE TITLE
Feature Request: Provision to categorize a set of tasks

### ISSUE THREAD

**Issue by NitinNair89** [varunherlekar/lets-do-it#33]
A user may want to create multiple to-do tasks and list them as per their categories. For example, a user can have two categories such as:

1. Shopping
2. Study

The "Shopping" category may have tasks/to-do items like "buy milk", "buy groceries", "buy clothes", etc. whereas the "Study" category may have tasks/to-do items like "complete assignment", "solve math", "meet up for group study", etc.

@varunherlekar what are your thoughts on this?

**Comment by varunherlekar** [comment]
Amazing

**Comment by NitinNair89** [comment]
Yeah, so this would mean you would need an extra form field, probably a `<select>` or a `<datalist>` element to accept categories from the user.

**Comment by varunherlekar** [comment]
Yes `select` or `datalist` will work.
